### ISSUE TITLE
npmjs.com readme says redirectSSL, should be redirect-ssl

### ISSUE THREAD

**Issue by tbredin** [unjs/redirect-ssl#38]
Nothing major here, in the [npmjs.com docs](https://www.npmjs.com/package/redirect-ssl) in the nuxt config section it says 

```
import redirectSSL from 'redirectSSL';
```

Which will throw an error. It should instead be:

```
import redirectSSL from 'redirect-ssl';
```

**Comment by danielroe** [comment]
Thanks for noticing! Fixed in https://github.com/unjs/redirect-ssl/pull/33 and npm doc will be updated on next release.

**Comment by pi0** [comment]
Thanks! Released in 2.0.1
